controllers/pwa.js

```diff
@@ -73,7 +73,10 @@ router.get('/add', (req, res) => {
  * Create a PWA.
  */
 router.post('/add', (req, res, next) => {
-  const manifestUrl = req.body.manifestUrl;
+  let manifestUrl = req.body.manifestUrl;
```

Comment:
Very minor, but might be good to trim whitespace before matching on `http://` (I think the whitespace will otherwise get through):

```javascript
let manifestUrl = req.body.manifestUrl.trim();
```